Validation/python/tagAndProbeDumperConfig.py

```diff
@@ -0,0 +1,199 @@
+import re
```

Comment:
This files contains way more than it's needed to run a clean t&p, can you please rearrange it?